packages/primitives/radio/readme.md

```diff
@@ -1,5 +1,36 @@
 `Radio` brings the equivalent of an `<input type="radio" />` to Native land.
 
-## Handling changes
+## Basic usage
 
-Context passes down a `groupValue` and a `groupName` together with a `setGroupValue` action to all its consumers. Both the value and the state are set by a `useState` hook within the consumer’s side, while `groupName` is a hardcoded `string` that marks the group name.
+```jsx
```

Comment:
i would still add some explanation, like: "instead of using `checked` we are comparing groupValue to value" and also maybe also the "Both the value and the state are set by a `useState` hook within the consumer’s side, while `groupName` is a hardcoded `string` that marks the group name." even though i'm not completely sure what does it mean to use the "`useState` hook"